iam.tf

```diff
@@ -254,39 +254,43 @@ resource "aws_iam_role_policy" "lambda" {
   role   = aws_iam_role.lambda[0].id
 }
 
+data "aws_partition" "current" {}
+
+data "aws_region" "current" {}
```

Comment:
The region is already looked-up in `main.tf`:

```hcl
# Find the region.
data "aws_region" "default" {}
```

Please remove this (`current`) one and refer to the current region using: `data.aws_region.default.name`.

Reply:
Done, has been fixed in commit [ded31db](https://github.com/robertdebock/terraform-aws-vault/pull/60/commits/ded31db9d27455426a090f09cf8770505ccccdff)